styles.css

```diff
@@ -615,7 +614,163 @@
   display: none;
 }
 
+/* popup */
+.popup-details {
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  backdrop-filter: blur(7px);
+  z-index: 10;
+  justify-content: center;
+  overflow-wrap: break-word;
+}
```

Comment:
- [ ] Hi @ngeelani48, great effort working on the modal pop-up as suggested by the previous reviewer, but while making those efforts, I noticed there is some error you left in your code that needs to be fixed. Below are highlights of where your attention is needed.

   - The image link of the `1st` and `4th` project cards is broken when the pop-up is displayed. And as seen in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=zSbl2S8jSTfanLgf-0) of the template you choose, The image on the `1st` and `4th` card is not missing in the project main page. kindly see to this
   - The `see live` and `see source` icon link is broken and the buttons are not properly aligned as we have it in the Figma design
   - Currently in your design, Once the pop-up modal is displayed the closing icon (X) is not showing on the top-right-hand corner which makes it difficult for users to close the displayed pop-up.
   - The Images of the other cards that are showing when the pop-up is displayed are not properly aligned and it is not occupying the required space it is supposed to occupy as seen in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=zSbl2S8jSTfanLgf-0).

> N.B: Please Focus your attention on both the `desktop` and `mobile` versions of your project.

Your Design 👇 

| Desktop  | Mobile |
| ------------- | ------------- |
| ![naf pop](https://user-images.githubusercontent.com/78634841/216123302-1f1e2c5d-098a-47af-85bf-ae1b6d7b314f.png) | ![image](https://user-images.githubusercontent.com/78634841/216124351-6d549df9-c2c0-4da2-b454-27e05f09c56b.png) |

Figma Design 👇 

| Desktop  | Mobile |
| ------------- | ------------- |
|![pp2-real](https://user-images.githubusercontent.com/78634841/216123245-fa67671a-333d-4682-80cd-5debb9024e86.png)  | ![pp2-mob](https://user-images.githubusercontent.com/78634841/216123562-47180929-b586-4641-a9cc-259d592e5d7d.png)  |

Reply:
@brainconnect93 thanks for your kind review 👍 
Kindly these problems you mentioned are not showing on my side. have a look at the pictures attached.
![desktop](https://user-images.githubusercontent.com/117263981/216128653-a9fea5ed-0f41-4f76-a22a-73aa4990e788.png)
![mobile](https://user-images.githubusercontent.com/117263981/216128668-077cd05a-a39d-4652-acc1-397c1aa7e8ff.png)

Reply:
Hi @ngeelani48, It wasn't showing that way from my end, Kindly check the directory of the image you are targeting and be sure it is correct. Kindly resubmit for a second opinion and if it works fine on other reviewers' screens am very sure your project will be approved.
Thanks